### PR TITLE
Patch release of the azure-communication-identity [1.4.4]

### PR DESCRIPTION
**Pull request by maximrytych-ms**
# Description

Patch release of the azure-communication-identity [1.4.4]

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/communication/azure-communication-identity/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Upgraded `azure-core` from `1.36.0` to version `1.37.0`.
```
Comment (mjafferi-msft) [inline sdk/communication/azure-communication-identity/CHANGELOG.md:10]: Also mention the azure-communication-common version upgrade (from 1.2.5 to 1.2.6 I guess)

Comment (AikoBB) [reply sdk/communication/azure-communication-identity/CHANGELOG.md:10]: and identity one as well

Comment (maximrytych-ms) [reply sdk/communication/azure-communication-identity/CHANGELOG.md:11]: Done.

File: sdk/communication/azure-communication-identity/pom.xml [hunk 2]
```diff
@@ -132,6 +132,12 @@
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
         <version>0.8.8</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
+        <configuration>
```
[the remark this answers is not in the document]
Comment (maximrytych-ms) [reply sdk/communication/azure-communication-identity/pom.xml]: Removed.

**Review by mjafferi-msft — APPROVED** [review]

**Review by vcolin7 — APPROVED** [review]

**Pull request merged**